frontend/app/src/store/tasks/index.ts

```diff
@@ -169,6 +190,18 @@ export const useTaskStore = defineStore('tasks', () => {
           } else if (result === null) {
             let errorMessage: string;
             if (message) {
+              if (message === USER_CANCELLED_TASK) {
+                if (checkIfDevelopment() && !import.meta.env.VITE_TEST) {
```

Comment:
maybe put the logger here.
and in other task caller, in the `catch part`, just do:

`if (!(e instanceof UserCancelledTaskError)) {...`

Reply:
okay, I see, makes sense.